src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -575,7 +565,7 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
             return createCompilationLambda();
         }
 
-        public CompilationVerifier CompileWithCustomILSource(string cSharpSource, string ilSource, Action<CSharpCompilation> compilationVerifier = null, bool importInternals = true, string expectedOutput = null)
+        public CompilationVerifier CompileWithCustomILSource(string cSharpSource, string ilSource, Action<CSharpCompilation> compilationVerifier = null, bool importInternals = true, string expectedOutput = null, TargetFramework targetFramework = TargetFramework.Standard)
```

Comment:
Idk what's going on with `cSharpSource` but it offends me and I don't like looking at it.

Reply:
Fixed in #24962